### PR TITLE
[es] Translating sentences from #1986 into Spanish

### PR DESCRIPTION
**Pull request by davefx**

### CODE REVIEW

File: sentences/es/media_player_HassMediaNext.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ intents:
   HassMediaNext:
     data:
       - sentences:
-          - "[<reproduce>] ([<pista> ]siguiente|siguiente <pista>) [en|para] <name>"
+          - "[<reproduce>] [el|la] ([<pista> ]siguiente|siguiente <pista>) [en|para] <name>"
```
Comment (cibernox) [inline sentences/es/media_player_HassMediaNext.yaml:6]: `para`? That seems weird, I can't think on a grammatically correct way of using `para` in that position.

Comment (cibernox) [reply sentences/es/media_player_HassMediaNext.yaml:6]: nor there are any examples of that in the tests

**Comment by davefx** [comment]
"Siguiente vídeo para la televisión del salón"
--
David Marín Carreño ***@***.***>

**Comment by cibernox** [comment]
@davefx sounds pretty bad to me, but I'll allow it.

**Comment by frenck** [comment]
Because there hasn't been any activity on this PR for quite some time now, I've decided to close it for being stale.

Feel free to re-open this PR when you are ready to pick up work on it again 👍

../Frenck

**Pull request closed without merging**